### DISCUSSION TITLE
Is it necessary to specify the language for best results?

### DISCUSSION THREAD

**Discussion by qo4on** [openai/whisper#2376] [Q&A]
Does selecting a specific language such as `--language en`, `--language es`, `--language ru` and others instead of automatically selecting `--language None` affect the quality of transcription?

**Comment by ryanheise** [comment]
There should be no difference in quality assuming Whisper detects the correct language. But if it detects the wrong language, that will affect the transcription quality in the same way as if you had specified the wrong language with `--language wrong_language...`.

**Comment by qo4on** [comment]
I noticed that if you specify wrong language `--model turbo --language wrong_language`, the quality of transcription decreases, also new lines are added in text files where they should not be. Did I get it right, is this how it should be?

**Reply by ryanheise** [reply]
Yes, specifying the wrong language is only going to produce confusing results.
